judgments/views/detail.py

```diff
@@ -110,11 +110,27 @@ def detail(request, document_uri):
         else reverse("detail_pdf", args=[document.uri])
     )
 
+    if document.document_noun == "press summary":
+        breadcrumbs = [
+            {
+                "url": "/" + context["linked_document_uri"],
```

Comment:
why do we need url in the case of press summaries?

Reply:
This is keeping behaviour the same - at the moment it's structured:

Find Case Law > The Case of Foo vs Bar > Press Summary

Reply:
I had forgotten the fact that the middle section needs the url for it to be a link